library/src/scala/compiletime/package.scala

```diff
@@ -27,13 +29,19 @@ package object compiletime {
    *  }
    *  logged(identity("foo"))
    *  // above is equivalent to:
-   *  // ("code: identity("foo")", identity("foo"))
+   *  // ("code: scala.Predef.identity("foo")", identity("foo"))
    *  ```
    *
    * @note only by-name arguments will be displayed as "code".
    *       Other values may display unintutively.
    */
-  inline def (self: => StringContext) code (args: => Any*): String = ???
+  transparent inline def (inline self: StringContext) code (inline args: Any*): String = ${ codeExpr('self, 'args) }
```

Comment:
It is not clear to me why we need `transparent`, given that the result type is always `String`.

Reply:
It is because we want this to return a string literal that can be constant folded with other string literals. Also, the `error` method used the type of the string know the value.

Reply:
The intrinsic implementation was also transparent, but we forgot to mark the definition as such.